src/store/modules/farmer/selectors.ts

```diff
@@ -0,0 +1,26 @@
+import { createSelector } from 'reselect';
+import { Farmer } from './types';
+import { State } from '../../types';
+
+const getFarmerEntries = (state: State) => state.farmer.farmers;
+// const getCurrentFarmerUUID = (state: State) => state.currentFarmer.currentFarmerUUID; //need to merge with dairy/master for this
+
+/************Selectors for a specific farmer ***************/
+
+/**Selector for all farmers in the store */
+export const getAllFarmers = createSelector(
+  [getFarmerEntries],
+  (farmerEntries: Farmer[]) => farmerEntries);
+
+/**Selector for a subset of fields for all farmers (used on search page) */
+export const getFormattedFarmers = createSelector(
+  [getFarmerEntries],
+  (farmEntries: Farmer[]) => farmEntries.map(entry => 
+    ({ name: (entry.firstName + ' ' + entry.lastName), phoneNumber:('(' + entry.phoneArea + ') ' + entry.phoneNumber), uuid:entry.uuid }), 
```

Comment:
I would recommend looking into string templating for this, might make your job easier

For example `entry.firstName + ' ' + entry.lastName`

could be written as `'${entry.firstName} ${entry.lastName}'` using backticks instead of the typical single quotation